scripts/deploy_lockup.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -e
+
+if [ -z "${NODE_ENV}" ]; then
+  echo "NODE_ENV is required, e.g. \`export NODE_ENV=testnet\`"
+  exit 1
+fi
+
+if [ -z "${MASTER_ACCOUNT_ID}" ]; then
+  echo "MASTER_ACCOUNT_ID is required, e.g. \`export MASTER_ACCOUNT_ID=near\`"
+  exit 1
+fi
+
+echo "Using NODE_ENV=${NODE_ENV}"
+echo "Using MASTER_ACCOUNT_ID=${MASTER_ACCOUNT_ID}"
+
+# Verifying master account exist
+RES=$(near state $MASTER_ACCOUNT_ID | grep "amount" && echo "OK" || echo "BAD")
+if [ "$RES" = "BAD" ]; then
+  echo "Can't get state for ${MASTER_ACCOUNT_ID}. Maybe the account doesn't exist."
+  exit 1
+fi
+
+read -p "Enter account ID (prefix) to create: " ACCOUNT_PREFIX
+
+PREFIX_RE=$(grep -qE '^([a-z0-9]+[-_])*[a-z0-9]+$' <<< "$ACCOUNT_PREFIX" && echo "OK" || echo "BAD")
+
+if [ "$PREFIX_RE" = "OK" ]; then
+  ACCOUNT_ID="$ACCOUNT_PREFIX.${MASTER_ACCOUNT_ID}"
+else
+  echo "Invalid new account prefix."
+  exit 1
+fi
+
+
+LOCKUP_ACCOUNT_ID="lockup.$ACCOUNT_ID"
```

Comment:
we are planning to do instead `lockup` account that we control, and deploy under `$ACCOUNT_ID.$LOCKUP_ID` because otherwise we can't actually deploy this undert this account as we don't have perm

Reply:
make sense